packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -1031,6 +1026,33 @@ export class ProjectService extends BaseService {
         return { jobUuid: job.jobUuid };
     }
 
+    static getAnalyticProperties(
+        createProject: CreateProject,
+        projectUuid: string,
+        user: SessionUser,
+        method: RequestMethod,
+    ): ProjectEvent['properties'] {
+        const warehouseType = createProject.warehouseConnection.type;
+        const authenticationType =
```

Comment:
Do the other warehouses only have a single form of auth?

Reply:
Yes, check `common/src/types/projects.ts`